TDAT: / * public void test save reparacio ( ) throws exception {  <NL> system . out . println ( " save reparacio " ) ;  <NL> nota reparacio nota reparacio = null ;  <NL> reparacio controller instance = null ;  <NL> instance . save reparacio ( nota reparacio ) ;  <NL>  / / todo review the generated test code and remove the default call to fail .  <NL> fail ( " the test case is a prototype . " ) ;  <NL>  } * /  COM: <s> test of save reparacio method of class reparacio controller </s>